broker/src/main/java/io/moquette/broker/unsafequeues/QueuePool.java

```diff
@@ -384,6 +384,23 @@ public Queue getOrCreate(String queueName) throws QueueException {
         }
     }
 
+    void purgeQueue(String name) {
+        final QueueName queueName = new QueueName(name);
+        final LinkedList<SegmentRef> segmentRefs = queueSegments.remove(queueName);
+        SegmentRef segmentRef = segmentRefs.pollLast();
+        segmentsAllocationLock.lock();
+        try {
+            while (segmentRef != null) {
+                LOG.debug("Consumed tail segment {} from queue {}", segmentRef, queueName);
```

Comment:
Would rewrite this log as

```suggestion
                LOG.debug("Purging segment {} from queue {}", segmentRef, queueName);
```

Reply:
No problem, I added both logging lines.